style.css

```diff
@@ -66,7 +66,6 @@ a {
   height: 192px;
   margin: 0;
 }
-
 .headline p {
   font-family: Inter, sans-serif;
```

Comment:
Everything is good so far 💪 , on a slight changes to work on. To avoid mistakes in escaping, it is recommended to quote font family names that are generic.